sympy/matrices/tests/test_matrices.py

```diff
@@ -213,6 +213,13 @@ def test_power():
         [a**n, a**(n-1)*n, 0],
         [0, a**n, 0],
         [0, 0, b**n]])
+    assert Matrix([[3, 0, 0, 0, -3], [0, -3, -3, 0, 3],
+        [0, 3, 0, 3, 0], [0, 0, 3, 0, 3], [3, 0, 0, 3, 0]])**10 == Matrix([
+        [0, -59049, 59049, 0, 177147],
+        [-59049, -59049, 0, -59049, 118098],
+        [-177147, -177147, 0, -295245, 177147],
+        [59049, 236196, -118098, 236196, -413343],
+        [-177147, -177147, 0, -236196, 236196]])
 
```

Comment:
Don't use commas, call the private methods directly. We separated the code in order to test it independently.